models/forum.js

```diff
@@ -28,14 +28,48 @@ var ForumSchema = new Schema({
 		trim: true,
 		required: 'A thread needs the id of the poster'
 	},
-	imgurl: {
+	imgURL: {
 		type: String,
 		trim: true
 	},
-	comments: { // todo: validate inserted comments?
-		type: Array,
+	comments: {
+		type: [String],
 		default: []
-	}
+	},
+	edited: {
```

Comment:
Would we like a date indicating the datetime when something as edited?

Reply:
That would be a better strategy indeed.